brave-core/src/main/java/com/github/kristofa/brave/FlushingSpanCollector.java

```diff
@@ -70,7 +71,14 @@ public void flush() {
   /** Calls flush on a fixed interval */
   static final class Flusher implements Runnable {
     final Flushable flushable;
-    final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
+    final ScheduledExecutorService scheduler = Executors.newSingleThreadScheduledExecutor(new ThreadFactory() {
+      @Override
+      public Thread newThread(final Runnable r) {
+        final Thread t = new Thread(r, "brave-core-flushing-span-collector-flusher");
```

Comment:
I'd simplify this to the enclosing classname `FlushingSpanCollector` as it isn't likely there's a naming conflict, and using the classname makes lookups easier when there's a problem.